__tests__/integration/placeholder.int-test.ts

```diff
@@ -0,0 +1,5 @@
+describe('Integration test placeholder', () => {
```

Comment:
To verify our testing environment setup, and following Martin's recommendation from the last standup about testing best practices, it might be helpful to implement a basic integration test related to Firestore. This will help ensure our setup is configured correctly.